src/content/contributing/release-process/_index.en.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Release process"
+date: 2020-03-18T16:03:26+01:00
+weight: 10
+---
+
+This section explains the necessary steps to make a submariner release.
+It is assumed that you are familiar with the submariner project and the various repositories.
+
+
+# Step 1: create a submariner release
+
+
+Assuming that you have an existing submariner git directory, the following steps create a release named "Globalnet Overlapping IP support RC0" with version v0.2.0-rc0 based on the master branch.
+
+```bash
+cd submariner
+git remote add upstream ssh://git@github.com/submariner-io/submariner
+git fetch -a -v -t
+git checkout remotes/upstream/master -B master
+git tag -s -m "Globalnet Overlapping IP support RC0" v0.2.0-rc0
+git push upstream v0.2.0-rc0
+```
+
+A tagged release should appear [here](https://github.com/submariner-io/submariner/tags).
+
+> https://github.com/submariner-io/submariner/tags
```

Comment:
I think this is helpful when looking at the result, because otherwise the real link is hidden and you can't see it . It's for developers and not final users. Just a subjective detail btw ;)

Reply:
sure